kolibri/auth/management/commands/fullfacilitysync.py

```diff
@@ -30,7 +34,7 @@ def add_arguments(self, parser):
 
     def get_dataset_id(self, base_url, dataset_id):
         # get list of facilities and if more than 1, display all choices to user
-        facility_url = urljoin(base_url, 'api/facility/')
+        facility_url = urljoin(base_url, 'api/public/v1/facility/')
```

Comment:
Can we not do a `reverse` on the url here?

Reply:
this is true!

Reply:
If there's anywhere we might *not* want to use URL reversal, it's on public API endpoints, since we specifically need to ensure they don't change, as the client and server may be running different versions. Doesn't hurt to use it here, but in tests we should just hardcode (I think that's what we do in Morango tests).